pkg/ddc/jindofsx/transform.go

```diff
@@ -171,6 +171,18 @@ func (e *JindoFSxEngine) transform(runtime *datav1alpha1.JindoRuntime) (value *J
 	value.Master.DnsServer = dnsServer
 	value.Master.NameSpace = e.namespace
 	value.Fuse.MountPath = JINDO_FUSE_MONNTPATH
+	clusterDomain, err := common.GetClusterDomain()
+	// to catch get clusterDomain err
+	if err != nil {
+		switch {
+		case errors.Is(err, common.ErrCantFindResolvConf):
+			e.Log.Info("failed to parse cluster domain from resolv.conf")
+			err = nil
+			value.ClusterDomain = ""
```

Comment:
I am wondering if the error is another reason, how to handle it?

Reply:
I think the GetClusterDomain function should return the same error, and the process cannot fail because the file (/etc/resolve.conf) cannot be found, so I return the same error to GetClusterDomain with ErrCantFindResolvConf